expenses-manager-frontend/src/main/frontend/src/app/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,43 @@
+.sidebar{
+    width: 220px;
+    height: 100%;
+    background-color: #4F46E5;
+
+    .content{
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        @mixin font{
+            font-family: 'Outfit';
```

Comment:
Fonturile ar trebui prestabilite in styles.css pentru ca sunt ceva global si e redundant sa le declaram mereu in fisierele css.